tests/test_doctestplus.py

```diff
@@ -756,3 +756,83 @@ def f():
     reprec.assertoutcome(passed=1)
     assert reprec.listoutcomes()[0][0].location[0] == os.path.join('test', 'a', 'testcode.py')
     assert caplog.text == ''
+
+
+def test_remote_data(testdir):
+    testdir.makeini(
+        """
+        [pytest]
+        doctestplus = enabled
+    """)
+
+    p = testdir.makefile(
+        '.rst',
+        """
+        # This test should pass
+        .. doctest-remote-data::
+            >>> 1 + 1
+            2
+        """
+    )
+    testdir.inline_run(p, '--doctest-plus', '--doctest-rst',
+                       '--remote-data').assertoutcome(passed=1)
```

Comment:
It would be better to split the function in multiple tests.

Reply:
I'm not sure what you mean

Reply:
The test function has 5 different examples, those could be put in dedicated test functions so it's easier to run only one, or to find which one is failing (currently if the first one fails the others are not run).

Reply:
OK, so with super fresh eyes I make another review of this.

I would argue here, that I think we should have two inline runs for all there test functions, one with ``remote-data`` and one without, with changing the assert. What do you think @saimn? I would find that more easy to follow than two separate functions repeated after each other with a change only in the inline_run.

Reply:
I don't remember how it was done, but the current way with two `inline_run` calls seems fine :+1: